### ISSUE TITLE
AttributeError: 'LooseVersion' object has no attribute 'version'

### ISSUE THREAD

**Issue by Nomadjackalope** [bulletmark/libinput-gestures#137]
when running libinput-gestures -d I get the following error

```
[Errno 2] No such file or directory: 'libinput-list-devices'
Traceback (most recent call last):
  File "/usr/bin/libinput-gestures", line 83, in <module>
    if Version(libvers) >= Version('1.8'):
  File "/usr/lib/python3.5/distutils/version.py", line 70, in __ge__
    c = self._cmp(other)
  File "/usr/lib/python3.5/distutils/version.py", line 335, in _cmp
    if self.version == other.version:
AttributeError: 'LooseVersion' object has no attribute 'version'
```

**Comment by bulletmark** [comment]
Please add some explanation why you raised and then immediately closed this?

**Comment by Nomadjackalope** [comment]
I started writing this while working through troubleshooting my issue and accidentally pressed enter so it posted and then I edited it to have all the info.

I got this error because I didn't follow the directions properly. 

I missed

`sudo apt-get install libinput-tools`

After I found that and it worked, I closed this.
Apologies for the errant post.

**Comment by bulletmark** [comment]
But it seems like a valid bug! You should never get an exception error like that just because you didn't have that package installed.  I don't have access to a Debian or Ubuntu system so I will have to try and simulate this.

**Comment by bulletmark** [comment]
Fixed by commit 3fe6aec4a4f318eba5daf29943c29bc9b12a9dcd
